contract_wrapper/KlerosWrapper.js

```diff
@@ -53,8 +53,16 @@ class KlerosWrapper extends ContractWrapper {
   getDisputesForUser = async (
     account = this._Web3Wrapper.getAccount(0)
   ) => {
-    const profile = await this._StoreProvider.getUserProfile(account)
-    if (!profile) throw new Error("No user profile for " + account)
+    let profile
+    try {
+      profile = await this._StoreProvider.getUserProfile(account)
```

Comment:
I think it's not good to create the user in the catch if he don't exists.
I thin this way is better:
```
let profile = await this._StoreProvider.getUserProfile(account)
if (_.isNull(profile)
  profile = await this._StoreProvider.newUserProfile({
    address: account
  })
```